inference-engine/thirdparty/clDNN/src/gpu/detection_output_cpu.cpp

```diff
@@ -184,7 +184,11 @@ struct detection_output_cpu : typed_primitive_impl<detection_output> {
                    const bool share_location,
                    std::map<int, std::vector<int>>& indices,
                    std::vector<std::pair<float, std::pair<int, int>>>& scoreIndexPairs) {
-        std::sort(scoreIndexPairs.begin(), scoreIndexPairs.end(), comp_score_descend<std::pair<int, int>>);
+        std::sort(scoreIndexPairs.begin(),
+                  scoreIndexPairs.end(),
+                  [](const std::pair<float, std::pair<int, int>>& p1, const std::pair<float, std::pair<int, int>>& p2) {
+                      return (p1.first > p2.first) || (p1.first == p2.first && p1.second.second < p2.second.second);
```

Comment:
comp_score_descend is fixed according to this lambda function, but why not using that?

Reply:
fixed